scripts/s3-dist.sh

```diff
@@ -25,17 +25,20 @@ BUCKET_PREFIX=$1
 BUCKET_REGION=$2
 
 function cleanup(){
```

Comment:
weird not used to shell scripts you don't pass in a param, but instead use $1?

Reply:
$1 within the scope of `cleanup` is different from the scope of the rest of the script because a param is passed to the `trap` call [here](https://github.com/awslabs/fhir-works-on-aws-deployment/blob/5635531ad79cdf0fbfcf50546b71999f392b8f58/scripts/s3-dist.sh#L41)

Reply:
Interesting TIL :) more reading I checked into: https://www.shellscript.sh/functions.html